crates/sui-core/src/streamer.rs

```diff
@@ -4,15 +4,16 @@
 use crate::event_handler::EVENT_DISPATCH_BUFFER_SIZE;
 use futures::Stream;
 use mysten_metrics::spawn_monitored_task;
+use parking_lot::RwLock;
 use std::collections::BTreeMap;
 use std::fmt::Debug;
 use std::sync::Arc;
 use sui_json_rpc_types::Filter;
 use sui_types::base_types::ObjectID;
 use sui_types::error::SuiError;
 use tokio::runtime::Handle;
```

Comment:
nit: you might need to remove line 13 since it's unused now

Reply:
Good catch :)